### DISCUSSION TITLE
can we send new code in 3rd phase?

### DISCUSSION THREAD

**Discussion by sibylla00** [PaddlePaddle/PaddleSpatial#253] [BaiduKDDCup2022] [answered]
In the coming 3rd phase, can we send new code? or is it just the evaluation time then the participants can't renew the code? Thanks.

**Comment by ramber1836** [comment]
Yes, in Phase 3, new submission is allowed.

**Reply by sibylla00** [reply]
Thanks

**Comment by zhoujingbo** [comment, marked as answer]
From July 16th 00:01 AM to July 18th 11: 59 PM (AOE time), we will go to Phase 3 of the Baidu KDD Cup. During this phase, the test data will be updated. Each team can submit its prediction model three times per day. During this phase, the organizer team will not answer any questions about submission errors (except for the Pending Timeout error) in the discussion forum. The leaderboard in Phase 3 will determine the candidate awardee teams.

**Reply by sibylla00** [reply]
Thanks
